dynamic_array/include/dynamic_array/array.h

```diff
@@ -0,0 +1,305 @@
+#ifndef CPP_DYNAMIC_ARRAY_H_
+#define CPP_DYNAMIC_ARRAY_H_
+
+#include <iostream>
+
+namespace td {
+
+static const int kMinCapacity = 16;
```

Comment:
k_min_capacity.

constexpr

But, what is k?

Reply:
`konstant` - German mathematicians